gulpfile.babel.js

```diff
@@ -154,10 +160,52 @@ function litmus() {
   var cdnURL = CONFIG.aws.url;
 
   return gulp.src('dist/**/*.html')
-    .pipe($.replace(/=('|")(\/?assets\/img)/g, "=$1"+ cdnURL +"/"+ CONFIG.common.subject))
+    .pipe($.replace(/=('|")(\/?images)/g, "=$1"+ cdnURL +"/"+ CONFIG.common.subject))
     .pipe($.litmus(CONFIG.litmus));
 }
 
+// Copy and compress into Zip
+function zip() {
+  var dist = 'dist';
+  var ext = '.html';
+
+  function getHtmlFiles(dir) {
+    return fs.readdirSync(dir)
+      .filter(function(file) {
+        var fileExt = path.join(dir, file);
+        var isHtml = path.extname(fileExt) == ext;
+        return fs.statSync(fileExt).isFile() && isHtml;
+      });
+  }
+
+  var htmlFiles = getHtmlFiles(dist);
+
+  var moveTasks = htmlFiles.map(function(file){
+    var sourcePath = path.join(dist, file);
+    var fileName = path.basename(sourcePath, ext);
+
+    var moveHTML = gulp.src(sourcePath)
+      .pipe($.rename(function (path) {
+        path.dirname = fileName;
+        return path;
+      }));
+
+    var moveImages = gulp.src(sourcePath)
+      .pipe($.htmlSrc({ selector: 'img'}))
```

Comment:
Can you please move this to an actual "Issue" and feel free to reference this commit if you feel there is a correlation.